sentinel/server.js

```diff
@@ -50,15 +50,15 @@ serverChecks.check(db.serverUrl)
     return config.init()
       .then(() => {
         if (!loglevel) {
-          logger.transports.Console.level = config.get('loglevel');
-          logger.debug('loglevel is %s.', logger.transports.Console.level);
+          transports.terminal.level = config.get('loglevel');
+          logger.info(`loglevel is ${transports.terminal.level}`);
```

Comment:
I think we should remove this - I don't think anyone uses `loglevel` and we have other ways to configure debug level now.